spec/index.bs

```diff
@@ -1556,6 +1556,12 @@ applications should generally request the "worst" limits that work for their con
         The maximum number of {{GPUVertexState/buffers}}
         when creating a {{GPURenderPipeline}}.
 
+    <tr><td><dfn>maxBufferSize</dfn>
+        <td>{{GPUSize64}} <td>[=limit class/maximum=] <td>256 * 1024 * 1024
```

Comment:
The following style would match maxStorageBufferBindingSize better, as well as the value that actually comes out of the API.
```suggestion
        <td>{{GPUSize64}} <td>[=limit class/maximum=] <td>268435456 bytes (256 MiB)
```
Though we could alternatively choose to write something like `256 * 1024 * 1024 bytes (256 MiB)`